tests/tests_pytorch/serve/test_servable_module_validator.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+import torch
+
+from pytorch_lightning import seed_everything, Trainer
+from pytorch_lightning.demos.boring_classes import BoringModel
+from pytorch_lightning.serve.servable_module_validator import ServableModule, ServableModuleValidator
+
+
+class ServableBoringModel(BoringModel, ServableModule):
+    def configure_payload(self) -> ...:
+        return {"body": {"x": list(range(32))}}
+
+    def configure_serialization(self):
+        class Tensor:
+            @staticmethod
+            def deserialize(x):
+                return torch.tensor(x, dtype=torch.float)
+
+            @staticmethod
+            def serialize(x):
+                return x.tolist()
+
+        return {"x": Tensor.deserialize}, {"output": Tensor.serialize}
```

Comment:
Actually, since we are not using the class, this could be two local functions

```suggestion
        def deserialize(x):
            return torch.tensor(x, dtype=torch.float)

        def serialize(x):
            return x.tolist()

        return {"x": deserialize}, {"output": serialize}
```